zkevm-circuits/src/state_circuit/lookups.rs

```diff
@@ -86,43 +101,28 @@ impl<F: Field> Chip<F> {
         }
     }
 
-    pub fn configure(meta: &mut ConstraintSystem<F>) -> Config {
+    pub fn configure(
+        meta: &mut ConstraintSystem<F>,
+        u8_table: UXTable<8>,
+        u10_table: UXTable<10>,
+        u16_table: UXTable<16>,
+    ) -> Config {
         let config = Config {
-            u8: meta.fixed_column(),
-            u10: meta.fixed_column(),
-            u16: meta.fixed_column(),
+            u8_table,
+            u10_table,
+            u16_table,
             call_context_field_tag: meta.fixed_column(),
         };
-        meta.annotate_lookup_any_column(config.u8, || "LOOKUP_u8");
-        meta.annotate_lookup_any_column(config.u10, || "LOOKUP_u10");
-        meta.annotate_lookup_any_column(config.u16, || "LOOKUP_u16");
+        u8_table.annotate_columns(meta);
+        u10_table.annotate_columns(meta);
+        u16_table.annotate_columns(meta);
         meta.annotate_lookup_any_column(config.call_context_field_tag, || {
             "LOOKUP_call_context_field_tag"
         });
         config
     }
 
     pub fn load(&self, layouter: &mut impl Layouter<F>) -> Result<(), Error> {
```

Comment:
Should we have something like `self.config.u8_table.load()` here?

Reply:
The different with previous is, ux_table is passed from outside, so the caller will in charge of its table load.
Here the chip load should only responsible for the column by itself.
However I found indeed, table should loaded in state_circuit/dev.rs when it's synthesis as a circuit. Fixed in this commit https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1427/commits/b8e6ad49897de69b2568555b60ffb1224174f906

Reply:
Found 3 issue after pr re-trigger ci check (since now we have light test back)
1. `N_U8_LOOKUPS` will complain cell not enough, due to I changed `24 -> 12`, and move half to `N_U16_LOOKUPS`. However we do not cell move to U16 lookup yet. so I will tune it in later PR. For now I set  `N_U16_LOOKUPS=0`, and clippy will complain, therefore adding `#[allow(clippy::reversed_empty_ranges)]` just to silent it.
2. need to adjust test degree for evm circuit, so I hardcode `1<<16` to claim minimum row for evm circuit unittest (otherwise it will failed)